common/fuzzer_utils.py

```diff
@@ -77,6 +120,17 @@ def validate(fuzzer):
         return False
 
 
+def get_fuzzer_from_config(fuzzer_config: dict) -> str:
+    """Returns the fuzzer of |fuzzer_config| for a non-variant fuzzer or returns
+    the name for a fuzzer variant."""
+    return fuzzer_config.get('name', fuzzer_config['fuzzer'])
```

Comment:
Do we enforce fuzzer attribute always exist ?

Reply:
They should always exist (unlike name).
I think `fuzzer_config.get('fuzzer')` would be bad. I think it's better to exception.
Sorry if this doesn't answer the question, I'm not sure I fully understood.